cmd/collectors/zapiperf/README.md

```diff
@@ -2,23 +2,34 @@
 
 ZapiPerf collects performance metrics from ONTAP systems using the ZAPI protocol. The collector is designed to be easily extendible to collect new objects or to collect additional counters from already configured objects. (The [default configuration](../../../conf/zapiperf/default.yaml) file contains 25 objects)
 
-This collector is an extension of the [Zapi collector](../zapi/README.md) with the major difference between that ZapiPerf collects only the `perf` subfamily of the ZAPIs. Additionally, ZapiPerf always calculates final values from deltas of two subsequent polls.
+This collector is an extension of the [Zapi collector](../zapi/README.md), with the major difference between that ZapiPerf collects only the `perf` subfamily of the ZAPIs. Additionally, ZapiPerf always calculates final values from deltas of two subsequent polls.
```

Comment:
This collector is an extension of the [Zapi collector](../zapi/README.md). The major difference between them is that ZapiPerf collects only the performance (`perf') ZAPIs. Additionally, ZapiPerf always calculates final values from the deltas of two subsequent polls.